tests/continuous/test_quantile_interval.py

```diff
@@ -0,0 +1,185 @@
+"""
+Contains unit tests for scores.probability.continuous.quantile_interval_score
+"""
+
+try:
+    import dask
+    import dask.array
+except:  # noqa: E722 allow bare except here  # pylint: disable=bare-except  # pragma: no cover
+    dask = "Unavailable"  # type: ignore # pylint: disable=invalid-name  # pragma: no cover
+
+import pytest
+import xarray as xr
+
+from scores.continuous import quantile_interval_score
+from scores.utils import DimensionError
+from tests.continuous import quantile_interval_score_test_data as qistd
+
+
+@pytest.mark.parametrize(
+    ("lower_qtile", "upper_qtile", "lower_fcst", "upper_fcst", "error_message"),
+    [
+        (0, 1, qistd.FCST_LOWER, qistd.FCST_UPPER, qistd.ERROR_MESSAGE_QTILE.format(0, 1)),
```

Comment:
It looks like all elements of FCST_LOWER are below FCST_UPPER. It might be good to have a test case with some overlap to ensure that logic depending on any() and all() are handled correctly. Apologies if this is done later in the test.

Reply:
Thanks, a test with overlap has been added.